libs/ballot-interpreter-nh/src/cli/convert/index.ts

```diff
@@ -169,22 +169,27 @@ export async function main(
     ),
   });
 
-  if (convertResult.issues.length > 0) {
-    io.stderr.write(convertResult.success ? 'warning: ' : 'error: ');
+  const { issues = [] } = convertResult.ok() ?? convertResult.err() ?? {};
```

Comment:
In what cases will we hit the 2nd fallback to `?? {}`? Or is this just included because Typescript doesn't know that one of `ok()` and `err()` is guaranteed to be non-nullish?

Reply:
Yeah, it's TS not being smart enough.